webapp/src/components/issue_attribute_selector.jsx

```diff
@@ -60,13 +61,30 @@ export default class IssueAttributeSelector extends PureComponent {
         }
     }
 
-    onChange = (options) => {
-        if (!options) {
+    loadOptions = async () => {
+        this.setState({isLoading: true});
+        const options = await this.props.load();
```

Comment:
We should handle the error case here, by setting the error in this component's state, and setting `isLoading` to false.